sandboxnu/annotator
# The upload button now uploads files.

Our application should now add any and all files locally uploaded!

To test:
- Obtain the secrets.properties and awsconfiguration.json files from me. These should never be added to the repository as they contain the IAM credentials for the Annotator application.
- Create a file in the app's directory with the voice recorder or through your IDE.
- Press the 'upload' UI button, confirm that no errors have been logged, and confirm that the file(s) have been uploaded to the bucket and no longer exist on device!

Edit: To find the files in the bucket, look into Sandbox's S3 buckets. The files will be found at /annotator-userfiles-mobilehub-466999836/annotated-data/*
